saleor/graphql/schema.graphql

```diff
@@ -14289,6 +14289,16 @@ type Mutation {
     input: GiftCardSettingsUpdateInput!
   ): GiftCardSettingsUpdate @doc(category: "Gift cards")
 
+  """
+  Close circuit breaker for provided app. 
+  
+  Requires one of the following permissions: MANAGE_SETTINGS.
+  """
+  closeCircuitBreaker(
```

Comment:
I also have some doubts about the naming. What do you think about name that explains a feature, not the design pattern? Would something along the line of reenabling temporarily disabled sync webhooks for an app be better for the end user?

Reply:
Changed to `reenableSyncWebhooks` for now. I provided more details in Patrys's comment on the top.